tools/active_vision/explore/explore.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
could this file go in fairo/droidlet/tools instead of fairo/tools?

Reply:
Moved these to fairo/droidlet/tools a few commits later in this stack (https://github.com/facebookresearch/fairo/pull/1166).